demo-data/src/main/resources/xml-demo-data/product-version/newcastle-demo-product-1-version-1.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<productVersion>
```

Comment:
I'm just a little bit curious - what is the argument for having DB setup in database?